src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -61,51 +55,6 @@ public void GivenThereAreNoSchools()
             //throw new PendingStepException();
         }
 
-        [Given("the following schools exist")]
-        public async Task GivenTheFollowingSchoolsExist(Table table)
-        {
-            var url = $"data/ed-fi/schools";
-            var schools = table
-                .Rows.Select(row =>
-                {
-                    var gradeLevels = JsonSerializer
-                        .Deserialize<List<string>>(row["gradeLevels"])
-                        ?.Select(descriptor => new GradeLevel(descriptor))
-                        .ToList();
-
-                    var educationOrgCategories = JsonSerializer
-                        .Deserialize<List<string>>(row["educationOrganizationCategories"])
-                        ?.Select(descriptor => new EducationOrganizationCategory(descriptor))
-                        .ToList();
-
-                    var schoolId = row["schoolId"] != null ? int.Parse(row["schoolId"]) : (int?)null;
-                    var nameOfInstitution = row["nameOfInstitution"];
-
-                    return new School(
-                        schoolId: schoolId,
-                        nameOfInstitution: nameOfInstitution,
-                        gradeLevels: gradeLevels,
-                        educationOrganizationCategories: educationOrgCategories
-                    );
-                })
-                .ToList();
-
-            var apiResponses = new List<IAPIResponse>();
-
-            foreach (var school in schools)
-            {
-                var data = JsonSerializer.Serialize(school);
-                _apiResponse = await _playwrightContext.ApiRequestContext?.PostAsync(
-                    url,
-                    new() { Data = data }
-                )!;
-                apiResponses.Add(_apiResponse);
-            }
-            foreach (var response in apiResponses)
-            {
-                response.Status.Should().BeOneOf([200, 201]);
-            }
-        }
```

Comment:
This is no longer necessary, the generic form is now used

- Given the system has these "[resource]"